specs/WebResourceRequestedWorkers.md

```diff
@@ -0,0 +1,146 @@
+WebResourceRequested events for workers
```

Comment:
If there's an existing API doc for WebResourceRequested already checked in, and you'd like to update that doc rather than create a new doc, that is also an option, but not required.